zstash/update.py

```diff
@@ -107,8 +114,17 @@ def setup_update() -> Tuple[argparse.Namespace, str]:
         help="Hard copy symlinks. This is useful for preventing broken links. Note that a broken link will result in a failed update.",
     )
     args: argparse.Namespace = parser.parse_args(sys.argv[2:])
-    if args.hpss and args.hpss.lower() == "none":
+
+    if not args.hpss or args.hpss.lower() == "none":
```

Comment:
Parentheses, as in `create`, would be good: `if (not args.hpss) or (args.hpss.lower()) == "none":`

Reply:
True.  I was relying upon the default ("not" applies only the the very next argument).  Also to the shortcut-pass where testing (A or B) never tests B when A is true, as it is unnecessary (useful when testing B might cause an exception.

I added the parentheses.

Reply:
> Also to the shortcut-pass where testing (A or B) never tests B when A is true, as it is unnecessary (useful when testing B might cause an exception.

Yes, the parentheses are only for human readers. They shouldn't affect the code at all.